.github/workflows/add-to-project.yaml

```diff
@@ -0,0 +1,25 @@
+name: Add issues and PRs to project
+
+on:
+  issues:
+    types:
+      - opened
```

Comment:
do we also want to include `reopened` for both this and PRs?

if we do, does the add-to-project action correctly dedupe so we don't end up with 2 rows in the project?

Reply:
I don't think it will be necessary, because once it is added to the project, closing the issue does not remove it from the project. (It just moves it to the Done column.)

Reply:
Good catch, though! I just turned on a different (much easier to configure) automation to ensure reopened issues are moved out of Done.

Reply:
I guess I am thinking about the fact that there exist a lot of issues already on a lot of these repos which could get reopened at any point (and won't have already been added to the project unless you are thinking of doing some backfill). If the point is to ensure that things don't slip through the cracks, then I want to make sure that we have our bases covered.

Reply:
we should also consider `transferred`. What happens if we have a repo that is transferred to the connectrpc org and it has open issues on it already?